modules/utxo-lib/src/bitgo/UtxoTransactionBuilder.ts

```diff
@@ -25,14 +26,29 @@ export class UtxoTransactionBuilder<T extends UtxoTransaction = UtxoTransaction>
     if (txb) {
       (this as any).__INPUTS = (txb as any).__INPUTS;
     }
+
+    if (prevOutputs) {
+      const txbInputs = (this as any).__INPUTS;
```

Comment:
This a bit hacky too but I will go away when we implement psbt. I guess this is the least hacky workaround for having the prevout scripts available for tx builder.